### DISCUSSION TITLE
Is there a way to scale all assets in a scene?

### DISCUSSION THREAD

**Discussion by SeanBannister** [mozilla/hubs#2736] [General Questions & Discussion] [answered]
I created a scene and now I've realized the scale is wrong, the user just feels to small when they walk around inside it. But there doesn't appear to be an effective way to resize the entire scene, I tried:

1. Creating a group placing all assets inside it and scaling the group, sure this scales everything but for elements like particles this isn't effective because the particle size is specified separate to the scale of the particle element.
2. I tried resizing the spawn point, this resizes the avatar when they spawn and almost works however when the user utilities emotes or inserts assets via the create page they're now the wrong size.

Manually resizing everything looks like the only option, a little painful once you have a few assets in a scene.

I originally posted this at https://github.com/mozilla/Spoke/issues/991

**Comment by InfiniteLee** [comment, marked as answer]
There is no feature to do this currently. Thank you for opening your issue in the spoke repo, that will allow us to consider it for being added in the future.
